package.json

```diff
@@ -27,7 +27,6 @@
       "!dist",
       "!build",
       "src",
-      "test",
```

Comment:
ah nice one. Maybe it just would be better to use `"!test"` to make it explicit.

Reply:
Good point, thank you!